core/hashfuncs.h

```diff
@@ -75,9 +76,26 @@ static inline uint32_t hash_djb2_one_float(float p_in,uint32_t p_prev=5381) {
 		uint32_t i;
 	} u;
 
-	// handle -0 case
```

Comment:
Yeah I replaced the comment in response to this message. It looks like github didn't pick up that this line was now changed.